python/lsst/ts/ATMCSSimulator/mcs_csc.py

```diff
@@ -357,7 +357,7 @@ def do_trackTarget(self, data):
                 )
             if np.any(np.abs(velocity) > self.max_velocity[0:4]):
                 raise salobj.ExpectedError(
-                    f"Magnitude of one or more target velocities "
+                    "Magnitude of one or more target velocities "
                     f"{velocity} > {self.max_velocity}"
```

Comment:
This is a bit funny, is it combining a normal string with a f-string? a bit weird...

Reply:
I agree, but this change is required by the most recent flake8, which requires that f strings contain {}.

Each string that is concatenated is a completely separate string.